test/e2e/tag.feature

```diff
@@ -0,0 +1,40 @@
+Feature: JavaScript Tag
+  In order to use Karma
+  As a person who wants to write great tests
+  I want to be able to run tests from the command line.
+
+  Scenario: Execute a test in PhantomJs with version, without JavaScript tag
+    Given a configuration with:
+      """
+      files = ['basic/plus.js', 'basic/test.js'];
+      browsers = ['PhantomJS'];
+      jsVersion = 1.8;
+      plugins = [
+        'karma-jasmine',
+        'karma-phantomjs-launcher'
+      ];
+      """
+    When I start Karma
+    Then it passes with:
+      """
+      ..
+      PhantomJS
+      """
+
+  Scenario: Execute a test in Firefox with version, with JavaScript tag
+    Given a configuration with:
```

Comment:
You can create a new file similar to what is in `basic/test.js`, e.g. `basic/version.js` and include it in the files here. That means that file will be run in the browser. Now in that file you can put some js in that reads the current html and checks for the version tag to be present, and fails if it's not present.